exercises/binary-search/test/test_binary_search.c

```diff
@@ -0,0 +1,91 @@
+#include "vendor/unity.h"
+#include "../src/binary_search.h"
+
+void test_single_element(void)
+{
+   int arr[] = { 6 };
+   size_t length = sizeof(arr) / sizeof(arr[0]);
+   TEST_ASSERT_EQUAL_UINT64(0, binary_search(6, arr, length));
+}
+
+void test_value_in_middle(void)
+{
+   int arr[] = { 1, 3, 4, 6, 8, 9, 11 };;
```

Comment:
Yep that's a typo, it's on most of the tests due to copy/pasting. I'll remove it.